target/linux/realtek/patches-5.10/302-clocksource-add-otto-driver.patch

```diff
@@ -0,0 +1,42 @@
+--- a/drivers/clocksource/Kconfig
++++ b/drivers/clocksource/Kconfig
+@@ -127,6 +127,18 @@ config RDA_TIMER
+ 	help
+ 	  Enables the support for the RDA Micro timer driver.
+ 
++config OTTO_TIMER
```

Comment:
I know it's a small annoying nitpick, but to stay consistent with other drivers, could you name it
```suggestion
+config REALTEK_OTTO_TIMER
```
like we have `REALTEK_OTTO_WDT` and `REALTEK_OTTO_SPI` etc?